homeworks/03-data-structure/coverage/clover.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`coverage` folder should be added to `.gitignore`. And don't need to store it in the codebase.